### ISSUE TITLE
allowing glob patterns in `Contrasts.ConditionList` and `DummyContrasts.Contrasts`

### ISSUE THREAD

**Issue by Remi-Gau** [bids-standard/stats-models#44]
As this allowed in the design matrix, it would be handy to have it for the defining what goes into the contrasts.

Ran into the issue when working on https://github.com/bids-standard/model-zoo/pull/14

One part of this tuto just does a very simple F test on the realignement parameters that are entered into the design matrix as `trans_?` and `rot_?` but then have to be entered in the contrasts as `rot_x`, `rot_y`, `rot_z`, `trans_x`...

**Comment by effigies** [comment]
I agree that these should be available for any place that uses variables.
